app.py

```diff
@@ -389,26 +422,30 @@ def initDCR(self):
             return None
     def setDCR(self, res):
         """
-        Callback to receive return value from initDCR. 
+        Callback to receive return value from initDCR. Any arguments are
+        ignored.
```

Comment:
Arguments aren't ignored here. `setDCR` is a callback for the threaded `initDCR` call. See https://github.com/decred/tinydecred/blob/7ef2ba2d11c36703c9fed1304e1db06a67fc8255/app.py#L158

The `initDCR` function returns a bool, so that's what `setDCR` will receive, and if it's false, an error is displayed and the wallet sync is aborted. The thing about this method is that it's not really part of the "public" API. It's only used as a callback function for one particular multi-threaded task. Since it's Python, it may even be appropriate to have the leading underscore `_setDCR`.